packages/arb-token-bridge-ui/src/components/TransferPanel/hooks/useRouteStore.ts

```diff
@@ -0,0 +1,22 @@
+import { create } from 'zustand'
+
+export type RouteType = 'arbitrum' | 'oftV2' | 'cctp'
+export type RouteContext = null
+export type SetRoute = (route: RouteType, context?: RouteContext) => void
+interface RouteState {
```

Comment:
What is the difference between `RouteState` vs `RouteContext`? it's a bit confusing since both technically mean the same thing. Is there a way to simplify this confusion?